### PR TITLE
add mirrors stats to stats page

### PR DESCRIPTION
**Pull request by tahabi**
[rt#6177](https://ocf.io/rt/6177)

mirrors_stats

Maybe some pretty line graphs in the future

### CODE REVIEW

**Comment by tahabi** [comment]
I have a cronjob on fallingrocks that should update ocfstats.mirrors with the latest data from the logs, modeled on @jvperrin's script. I'll puppet it eventually.

File: ocfweb/stats/mirrors.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            'start_date': current_semester_start,
+        }
+    )
+
+
+def _bandwidth_by_dist(start):
+    with get_connection() as c:
+        c.execute(
+            'SELECT `dist`, SUM(`up` + `down`) as `bandwidth` FROM `mirrors_public` WHERE `date` > %s'
+            'GROUP BY `dist` ORDER BY `bandwidth` DESC', (start))
```
Comment (chriskuehl) [inline ocfweb/stats/mirrors.py]: start isn't a tuple here, either add a comma (like `(start,)`) or drop the unnecessary parens

File: ocfweb/stats/mirrors.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    return [(i['dist'], naturalsize(i['bandwidth'])) for i in c]
```
Comment (chriskuehl) [inline ocfweb/stats/mirrors.py]: can we avoid adding another dependency for this and just write the function ourselves? here's the implementation i usually use:

```
def human_size(bytes):
    for label, n in (
        ('TB', 2**40),
        ('GB', 2**30),
        ('MB', 2**20),
        ('KB', 2**10),
    ):
        if bytes > n:
            return '{:.1f} {}'.format(bytes / n, label)
    else:
        return '{} B'.format(bytes)
```

File: ocfweb/stats/templates/stats/mirrors.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+{% load common %}
+{% load stats %}
+
+{% block content %}
+<div class="ocf-content-block">
```
Comment (chriskuehl) [inline ocfweb/stats/templates/stats/mirrors.html]: indent the stuff inside `{% block %}`?

**Pull request merged**